qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -444,6 +446,8 @@ def test_dynamic_model_load_unload(self):
 
                 version_status = ss.model_status[savedmodel_name].version_status[3]
                 self.assertEqual(version_status.ready_state, server_status.MODEL_UNAVAILABLE)
+                self.assertEqual(version_status.ready_state_info.message, "unloaded",
```

Comment:
Isn't this going for fail, should be ready_state_reason?

Reply:
Yes it should have been, I missed the change in tests... Will relaunch CI.